manifests/jail.pp

```diff
@@ -1,64 +1,149 @@
+# @summary Jail for an SFTP user.
+#
+# @example Single user
+#   sftp_jail::jail { 'myjail':
+#     user  => 'bbriggs',
+#     group => 'bbriggs',
+#   }
+#
+# @example Shared jail
+#   sftp_jail::jail { 'shared_jail':
+#     user        => 'writeuser',
+#     group       => 'writegroup',
+#     match_group => 'sftpusers',
+#   }
+#
+# @param jail_name
+#   The jail name.
+#
+# @param jail_base
+#   The jail's full path.
+#
+# @param user
+#   The main user that will own the corresponding home directory in the jail,
+#   giving the user a place to land. Also sets user ownership for `/incoming`.
+#
+# @param group
+#   The users group.
+#
+# @param groups
+#   Additional groups the user should be in. Only usefull for admins.
+#
+# @param  passowrd_hash
+#   generate password hash with salt:
+#     `python3 -c 'import crypt,getpass; print(crypt.crypt(getpass.getpass()))'`
+#
+# @param real_home
+#   The real full path users home.
+#
+# @param home
+#   The users $HOME environment variable.
+#
+# @param comment
+#   Users comment / GECOS.
+#
+# @param manage_user
+#   Manage the user resource?
+#
+# @param sftp_admin
+#   Is this user an SFTP admin? SFTP Admins can read an probaply write to user
```

Comment:
`probaply` looks like a typo

Reply:
correct, fixed in https://github.com/voxpupuli/puppet-sftp_jail/pull/69/commits/42c580f64ce9682bbeb3612fd2add003f5f9ff8a